spec/compiler/parser/to_s_spec.cr

```diff
@@ -108,4 +108,9 @@ describe "ASTNode#to_s" do
   expect_to_s %(1 <= 2 <= 3)
   expect_to_s %((1 <= 2) <= 3)
   expect_to_s %(1 <= (2 <= 3))
+
+  expect_to_s %(case\nwhen 1\nwhen 2\nend)
```

Comment:
It's `case`, not `case!`. There is no `case` case in `to_s_spec.cr`, so I added them also.